spec/factories/physical_rack.rb

```diff
@@ -0,0 +1,3 @@
+FactoryGirl.define do
+  factory :physical_rack
+end
```

Comment:
This should no longer be necessary... "default" factories are automatically build now.